app/views/layouts/printer_friendly.html.haml

```diff
@@ -1,11 +1,13 @@
 !!!
 %html{dir:language_direction}
-  %head
+  %head      
     %title
       =h html_title
     %style
       a.anchor, a.shy {display:none;}
+      @media print { .noPrint {display: none}}
   %body{class:language_direction}
+    %button{:onclick => 'window.history.back()', :class => 'noPrint'} back
```

Comment:
Oh.. and we probably want a translated 'back':
```haml
%button.no-print{:onclick => 'window.history.back()'}
  = :back.t
```
The Symbol#t is a crabgrass specific shortcut for I18n.t(symbol)